src/components/Player.js

```diff
@@ -15,8 +16,18 @@ export default function Player(props) {
     var internalPlayer = props.playerRef.current?.getInternalPlayer()
     var videoUrl = internalPlayer.getVideoUrl()
     if (!videoUrl) return
-    console.log(`BROKEN VIDEO: ${getVideoId(videoUrl)}`)
-    internalPlayer.nextVideo()
+
+    var videoId = getVideoId(videoUrl)
+    console.log(`BROKEN VIDEO: ${videoId}`)
+    Honeybadger.notify(`BROKEN VIDEO: ${videoId}`);
```

Comment:
This could potentially be quite noisy. But we can see how it goes. Also pretty often this value is just null for me, we can follow it and see what happens

Reply:
Example: 
`BROKEN VIDEO: null, videoUrl: https://www.youtube.com/watch`

Reply:
New Example: 

`BROKEN VIDEO: null, videoUrl: https://www.youtube.com/watch, video: {"title":"Owl City - Fireflies (Official Music Video)","video_id":"psuRGfAaju4","playlist_id":"PL96VlwoPgKeyYyH44K_6M423JbWdxp49G","playlist_owner":"Robert S"}`